modules/ROOT/pages/business-groups.adoc

```diff
@@ -0,0 +1,116 @@
+= Business Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Business groups provide a way to separate and control access to Anypoint Platform resources such as APIs and applications. Users have access only to the business groups to which they are added.  
+
+Business groups reside within the master organization and can contain their own child business groups. The master organization and its business groups are organized in a hierarchical tree, where the master organization is the root. Each business group has one direct parent and can have multiple children.
+
+[NOTE]
+Business groups are not enabled by default in a new Anypoint Platform account. To activate business groups in your organization, contact your MuleSoft representative. There is a limit of 100 business groups per organization.
+
+If you are an organization administrator and your organization is enabled for business group support, you can create business groups within your organization. The organization administrator is also the administrator of any business groups (sub-organizations) within the parent organization.
+
+[NOTE]
+There is a limit of 100 business groups per organization. 
+
+==== Child Business Groups
+
+Child business groups provide more fine grained control over access. 
+
+image::organization-2b25c.png[]
+
+Since business groups are hierarchical, the owner of a parent business group always retains administrator permissions for any child business group within that parent. This means that the owner of a business group can create child business groups and assign another user as the owner (owners automatically have the *Organization Administrators* role) of those child business groups. The owner of the parent group remains an *Organization Administrator* within the child business group. 
+
+Conversely, owners of child business groups cannot:
+
+* Access or modify the parent business group or master organization settings
+* View the parent business group's client ID and client secret
+
+When you create a child business group within a parent, only the vCores that were assigned to the parent business group can be allocated to the child business group.
+
+=== Create a Business Group
+
+Create business groups within your organization. When you create a new business group, all the users with the *Organization Administrators* role in the parent business group are also added as organization administrators in the newly created business group. However, when you create a new user as an *Organization Administrator* in the parent business group, the user is not added to child business groups.
```

Comment:
re: `the user is not added to child business groups.` -- I think it's extremely important that this section clarifies that you can't be "added" to a BG. You are always a member of the master org. Whether your user shows up in the list of users in the BG is dependent on roles and permissions, not "membership"